bin/sanity_tests

```diff
@@ -0,0 +1,7 @@
+source conf/drillTestConfig.properties
```

Comment:
May be call the file run_jdbc_sanity_tests?

Reply:
done